src/commands/drop.ts

```diff
@@ -0,0 +1,137 @@
+    confirm: flags.boolean({char: 'y', description: 'Skip Confirmation', default: false}),
+  }
+
+  confirm(message: string) {
+    this.log(message)
+    return cli.confirm('Are you sure you want to proceed?')
+  }
+
+  handleResult(errors: [{message: string}] | undefined, successMessage: string) {
+    if (errors) {
+      for (const {message} of errors) {
+        this.error(message)
+      }
+      return
+    }
+
+    this.log(successMessage)
+  }
+
+  async run() {
+    const opts = this.parse(DropData)
+    const backend = await this.backendFromOpts(opts)
+
+    if (opts.flags['list-unused']) {
+      const {data: graphqlex, errors: graphqlexErrors} = await backend.queryQuery<{data: object }>('schema { __typename }')
+      const {data: graphql, errors: graphqlErrors} = await backend.adminQuery<{getGQLSchema: { schema: string }}>('{ getGQLSchema { schema } }')
+
+      if (graphqlErrors || graphqlexErrors) {
+        this.handleResult(graphqlErrors, '')
+        this.handleResult(graphqlexErrors, '')
+        return
+      }
+
+      this.log('Unused Types: ', getTypesDiff(graphql.getGQLSchema.schema, graphqlex))
+      this.log('Unused Fields: ', getFieldsDiff(graphql.getGQLSchema.schema, graphqlex))
+      return
+    }
+
+    if (opts.flags['drop-schema']) {
+      if (opts.flags.confirm || await this.confirm('This will delete all data and schema in your backend, and cannot be reversed')) {
+        const {errors} = await backend.slashAdminQuery<{dropData: {response: {code: string; message: string}}}>(DROP_SCHEMA)
+        this.handleResult(errors, 'Successfully dropped all data and schema!')
+      } else {
+        this.log('Aborting')
+      }
+      return
+    }
+
+    if (opts.flags['drop-data']) {
+      if (opts.flags.confirm || await this.confirm('This will delete all data in your backend, and cannot be reversed')) {
+        const {errors} = await backend.slashAdminQuery<{dropData: {response: {code: string; message: string}}}>(DROP_DATA)
+        this.handleResult(errors, 'Successfully dropped all data!')
+      } else {
+        this.log('Aborting')
+      }
+      return
+    }
+
+    const deleteTypes = opts.flags['drop-types']
+    const deleteFields = opts.flags['drop-fields']
+    if (deleteTypes || deleteFields) {
+      if (!(opts.flags.confirm || await this.confirm('This will delete the listed unused types/fields, and cannot be reversed'))) {
+        this.log('Aborting')
+        return
+      }
+
+      if (deleteTypes) {
+        const {errors} = await backend.slashAdminQuery<{dropData: {response: {code: string; message: string}}}>(DROP_TYPES, {
+          types: deleteTypes,
+        })
+        this.handleResult(errors, 'Successfully dropped types: ' + deleteTypes)
+      }
+
+      if (deleteFields) {
+        const {errors} = await backend.slashAdminQuery<{dropData: {response: {code: string; message: string}}}>(DROP_FIELDS, {
```

Comment:
can you do this in a single query? Types and fields.

Reply:
This will require a change in the proxy. Will add as TODO and do it as part of other cleanup in proxy.